app/models/account_migration.rb

```diff
@@ -90,6 +93,16 @@ def user_changed_id_locally?
     old_user && new_user
   end
 
+  def identity_transplantation
```

Comment:
Maybe just `migrate!` or if we go for verbose `tombstone_old_user_and_update_all_references`. `transplantation` is not a term used elsewhere so it feels alien to me :)

Reply:
`migrate!` looks confusing, there are too many usages of the "migrate" word already. I thought for a while about something short and meaningful, but I couldn't invent anything good, so I'm for going verbose.